test/list-item.test.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import ListItem from '../src/list-item';
+
+describe('List item tests', () => {
+  const defaultProps = {
+    item: 'hello',
+  };
+  let wrapper = null;
+
+  beforeEach(() => {
+    wrapper = shallow(<ListItem {...defaultProps} />);
+  });
+
+  it('Should render with default cursor and li tag', () => {
+    expect(wrapper.exists()).toEqual(true);
+    expect(wrapper.find('li').exists()).toEqual(true);
+    expect(wrapper.prop('style')).toMatchObject({ cursor: 'default' });
+  });
+
+  it('Should have the pointer cursor if onClick exists', () => {
+    const newProps = {
+      ...defaultProps,
+      onClick: () => {},
+    };
+
+    wrapper = shallow(<ListItem {...newProps} />);
+
+    expect(wrapper.prop('style')).toEqual({ cursor: 'pointer' });
+  });
+
+  it('Should use dd tag if tag is dt', () => {
+    const newProps = {
+      ...defaultProps,
+      tag: 'dt',
+    };
+
+    wrapper = shallow(<ListItem {...newProps} />);
+
+    expect(wrapper.find('dd').exists()).toEqual(true);
+  });
+});
```

Comment:
Can we add a test to verify custom listItems works?

Reply:
yeah the test is in the List tests https://github.com/codeparticle/react-list/pull/2/files#diff-bf018b7d626373b883599463a950b3e4R29